src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -32,7 +32,8 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Authors R. W. Ford, A. R. Porter and S. Siso, STFC Daresbury Lab
-# Modified I. Kavcic Met Office, C.M. Maynard, Met Office/University of Reading.
+# Modified I. Kavcic Met Office
```

Comment:
Comma after "Kavcic" please.

Reply:
Changed (and added space between middle names for Duke Maynard).